pom.xml

```diff
@@ -42,8 +42,9 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <guava.version>20.0</guava.version>
-    <netty.version>4.1.10.Final</netty.version>
     <protobuf.version>2.6.1</protobuf.version>
+    <netty.version>4.1.10.Final</netty.version>
```

Comment:
@kishorekasi 
There are blocker issues reported on OpenSSL support on this version (4.1.10)
we should upgrade to 4.1.12 + 2.0.3final
http://netty.io/news/2017/05/12/4-0-47-Final-4-1-11-Final.html
https://netty.io/news/2017/06/09/4-0-48-Final-4-1-12-Final.html